### PR TITLE
Update to SQLDelight 2.0-alpha02

### PR DESCRIPTION
**Pull request by kevincianfarini**
This commit updates our version of SQLDelight to 2.0.
There's a lot of improvements for PostgreSQL, but we also
have to work around lots of early stage bugs.

The need to manipulate dates in #334 necessitates this PR.

### CODE REVIEW

**Review by EvanHahn — COMMENTED** [review]
Broadly: I see a dependency on an alpha version and I think "this might break something". I [see][0] that you're a contributor to SQLDelight so you probably have a good understanding of things, but I wonder if this is premature. Thoughts?

[0]: https://github.com/cashapp/sqldelight/graphs/contributors

**Comment by kevincianfarini** [comment]
Two things: 

I didn't want to update yet, but proper date support in PG necessitates this. 

I think using the alpha version is fine. The large majority of SQLDelight is actually a SQL compiler. That's where most of the 2.0 work has gone in, so i don't expect runtime bugs to bite us.

File: backend/src/main/sqldelight/org/climatechangemakers/act/database/FocusIssue.sq [hunk 1]
```diff
@@ -2,3 +2,6 @@ CREATE TABLE focus_issue (
   issue_id BIGINT NOT NULL REFERENCES issue ON DELETE CASCADE,
   focused_at TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP
 );
+
+selectAll:
+SELECT * FROM focus_issue;
```
Comment (kevincianfarini) [inline backend/src/main/sqldelight/org/climatechangemakers/act/database/FocusIssue.sq:7]: Remove.

**Comment by kevincianfarini** [comment]
@EvanHahn any reservations? This is blocking multiple PRs now because we really need proper date support.

**Review by EvanHahn — APPROVED** [review]
Sorry for the slow review. LGTM.

**Comment by kevincianfarini** [comment]
No worries :smile:

**Pull request merged**